tests/.env.example

```
@@ -20,3 +20,7 @@
 
 # Name of the bucket to store the evaluator results in
 # KAI_QE_S3_BUCKET_NAME="exampleBucket"
+
+# Default REPO used for testing
+# TEST_REPO_URL=https://github.com/konveyor-ecosystem/coolstore
+# TEST_REPO_NAME=coolstore
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)